src/module/item/physical/sheet.ts

```diff
@@ -105,7 +105,7 @@ class PhysicalItemSheetPF2e<TItem extends PhysicalItemPF2e> extends ItemSheetPF2
             return result;
         }, {} as MaterialSheetData["materials"]);
 
-        const value = material.precious ? `${material.precious.type}|${material.precious.grade}` : "";
+        const value = material.type && material.grade ? `${material.type}|${material.grade}` : "";
```

Comment:
I see that adamantine earlier in the PR defaults grade to standard if its unset. Should this do the same?

Reply:
That was to handle adamantine coming from a special ability, like Adamantine Strikes, which doesn't specify a grade.